### PR TITLE
Add Apple Crostata Recipe

### PR DESCRIPTION
**Pull request by zyansheep**
Found this on reddit, it looked really good...

### CODE REVIEW

**Comment by zyansheep** [comment]
why is the image limit 150k? It should be around 500kb because high-def images are cool.

File: src/apple-crostata.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+- 300-500ml Sweet Cream (depending on the volume of the baking dish)
+- 3 Eggs
+- 100g Sugar
+- ~3 tbsp Apricot Jam (optional)
+
+## Directions
+1. Mix the eggs by hand, stir in the sugar and slowly add the cream.
+2. Grease the baking tray well with butter and line it with puff pastry dough.
+3. Cut the apples into thin slices and arrange them in a baking dish however you like. Drizzle with cream and eggs.
+4. Bake in the oven at 425° F (215 °C) for 40 min.
```
Comment (hallzy) [inline src/apple-crostata.md:21]:
```suggestion
4. Bake in the oven at 425°F (215°C) for 40 min.
```

**Review by hallzy — CHANGES_REQUESTED** [review]
I hate to be 'that' guy, but can you shrink your image by at least 1062 bytes to make the CI script happy. :)

The original readme from Luke actually mentions less than 100kB so the 150 is being pretty generous relatively speaking

**Comment by zyansheep** [comment]
alright... I'll ffmpegify it. I committed it previously as it was because a lot of the existing images were way over 150kb

**Review by hallzy — APPROVED** [review]
Thanks :D

File: src/apple-crostata.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+![apple crostata](pix/apple-crostata.webp "Apple Crostata in Dish")
+
+- ⏲️ Prep time: 25-30 min
+- 🍳 Bake time: 40 min
+- 🍽️ Servings: 4 large slices
+
+## Ingredients
+- Puff Pastry Dough
+- 3-4 Apples
```
Comment (zyansheep) [inline src/apple-crostata.md:11]:
```suggestion
- 3-4 Peeled Apples
```
specify to use peeled apples

**Pull request closed without merging**